app/Http/Requests/Team/NewTeamRequest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Requests\Team;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class NewTeamRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'name' => ['required'],
```

Comment:
The team name should be unique, a string and it should be limited to 150 characters.
You are also missing validation for the the uploaded file. Make sure to add some validation for the file type (jpeg, png, svg and the max file size can be 1 MB), as well as minimum and maximum dimensions.

Reply:
Also, make sure that the logo is of type file.